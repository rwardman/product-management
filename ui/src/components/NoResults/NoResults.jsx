import React from "react";

import * as Styled from "./NoResults.styles";

const NoResults = () => {
  return (
    <Styled.Container>
      <h3>No products found</h3>
    </Styled.Container>
  );
};

export default NoResults;
