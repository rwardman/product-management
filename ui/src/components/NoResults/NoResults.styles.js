import styled from "styled-components";

const Container = styled.div`
  margin: auto;
`;

export { Container };
