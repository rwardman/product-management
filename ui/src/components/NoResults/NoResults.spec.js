import renderer from "react-test-renderer";
import NoResults from "./NoResults";

describe("When displaying the no results component", () => {
  it("should be displayed correctly", () => {
    const tree = renderer.create(<NoResults />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
