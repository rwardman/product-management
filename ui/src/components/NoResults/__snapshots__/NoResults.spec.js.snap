// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When displaying the no results component should be displayed correctly 1`] = `
<div
  className="sc-bdfBwQ fPZFRE"
>
  <h3>
    No products found
  </h3>
</div>
`;
